judge/admin/contest.py

```diff
@@ -153,6 +153,10 @@ def has_change_permission(self, request, obj=None):
             return True
         return obj.organizers.filter(id=request.profile.id).exists()
 
+    def _rescore(self, request, contest_key):
```

Comment:
Probably shouldn't be in a separate fcn if there's a single caller and it's this short